assessments/0008-backstage.md

```diff
@@ -0,0 +1,403 @@
+
+**Provide adequate navigation signals**
+
+Reorganizing the site will make the documentation more usable. Not to be overlooked is the companion task of making the documentation *findable*. This involves creating adequate tables of contents (TOCs), indexes, and a glossary to help navigate the site. Much of this is automated by the static site generator (currently *Docusaurus*), but it's the writer's responsibility to assure that these navigation aids are adequate.
+
+
+## Contributor documentation
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     |     |  :heavy_check_mark:  |    |
+| Beginner friendly issue backlog           |     |     |     |  :heavy_check_mark:  |     |
+| “New contributor” getting started content |     |     |  :heavy_check_mark:  |     |     |
+| Project governance documentation          |     |     |     |     |  :heavy_check_mark:  |
+
+Scale:
+- 1 = (Is not present or requires significant work)
+- 3 = (Is present, but needs work)
+- 5 = (Is executed extremely well or no improvement required)
+
+### Comments
+
+A unique problem with Backstage is that user and contributor roles exist on a spectrum; plugins designed or modified to serve a particular organization can be contributed to the Backstage project, or can originate there in anticipation of a particular organization's need. As a result, documentation for project contributors is intermingled, and easily confused, with instructions for admin users trying to expand functionality by modifying or adding code, especially plugins.
+
+#### Communication methods documented
+
+Communication channels are clearly documented on the [Community](https://backstage.io/community) page of the website, including **message channels**, **newsletters** and **adoption meetings**. There is some minor conflation of commercially sponsored content on the community page.
+
+The **[GitHub](https://github.com/backstage/backstage) repository** is directly linked from the main menu. There are 22 related repositories listed on the [Backstage project page](https://github.com/backstage). A little digging is required to suss out the purpose of some of these projects.
+
+Examples:
+https://kubernetes.io/community/
+
+#### Beginner friendly issue backlog
+
+The backlog is robustly **triaged** and documented. A **"good first issue"** label is available and prominently recommended. Issues are **well documented** with complete descriptions.
+
```

Comment:
We may want to take this opportunity to introduce the community to https://clotributor.dev/ (https://github.com/cncf/clotributor). It may be helpful for documentation as well as code contributions.

Reply:
Are you suggesting adding it to the assessment (done!) or to the assessment framework?

Reply:
It could go into either — it’s a new tool, and I thought of it while reading this, so made the suggested to include a mention to it. I think we could include it or not, but it’s one of those thing that works best with more visibility.